src/components/Select/Select.js

```diff
@@ -311,7 +321,19 @@ Select.propTypes = {
   /**
    * Warning or error message, displayed in a tooltip.
    */
-  validationHint: PropTypes.string
+  validationHint: PropTypes.string,
+  /**
+   * A string that will become the label for the input.
+   */
+  label: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+  /**
+   * Defines whether the label should be hidden or not.
+   */
+  labelVisuallyHidden: PropTypes.bool,
+  /**
+   * Sets the label for the input, needed for a11y.
+   */
+  id: PropTypes.string.isRequired
```

Comment:
Same suggestion as for the Input component :)

Reply:
updated